ext/opcache/ZendAccelerator.h

```diff
@@ -280,6 +283,8 @@ typedef struct _zend_accel_shared_globals {
 
 	/* Interned Strings Support (must be the last element) */
 	ZEND_SET_ALIGNED(ZEND_STRING_TABLE_POS_ALIGNMENT, zend_string_table interned_strings);
+
+	size_t map_ptr_static_last;
```

Comment:
You may not add this as the _last_ member because of the interned strings. Add it just before that.

Reply:
Oops, last minute change. Will change.

Reply:
Changed.